src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -147,9 +155,16 @@ public Object instanceMain(final String[] args) {
      * Note: this is the only method that is allowed to call System.exit (because gatk tools may be run from test harness etc)
      */
     protected final void mainEntry(final String[] args) {
-        final CommandLineProgram program = extractCommandLineProgram(args, getPackageList(), getClassList(), getCommandLineName());
+
+        // First we package our args together a little better:
+        ArrayList<String> argArrayList = new ArrayList<>( Arrays.asList(args) );
+
+        // Now we setup our configurations:
+        ConfigUtils.initializeConfigurationsFromCommandLineArgs(argArrayList, gatkConfigFileOption);
+
+        final CommandLineProgram program = extractCommandLineProgram(argArrayList.toArray(new String[] {}), getPackageList(), getClassList(), getCommandLineName());
```

Comment:
probably want to pull out the array as a new variable with a name to make it clear why we converted to list and then back instead of just using the existing one